Documentation~/cleanShotGrid.md

```diff
@@ -0,0 +1,17 @@
+## Uninstall existing ShotGrid Desktop apps
+
+Before installing a newer version of ShotGrid Desktop, it is recommended to remove
+the previously installed, older version, of ShotGrid Desktop.
```

Comment:
Replace "the previously installed, older version, of ShotGrid Desktop." with "the previous version"